package.json

```diff
@@ -21,7 +21,7 @@
     "prepublishOnly": "npm run build",
     "start": "node index.js",
     "test": "npm run lint",
-    "watch": "gulp watch"
+    "watch": "npx babel src --extensions .js --out-dir dist --verbose --watch"
```

Comment:
Nice catch on the unnecessary gulp dependency, but we also would not need `npx` in the case. NPM should be able to use the local babel dependency to run this.

Reply:
👍 just force-pushed removing `npx`